src/go/rpk/pkg/cli/cmd/container/common/common.go

```diff
@@ -442,3 +453,63 @@ This can happen for a couple of reasons:
 	}
 	return err
 }
+
+func NodeAddr(port uint) string {
```

Comment:
If `RenderClusterInfoStatus` lives in the container package, this does not need to exist here